scripts/get-all-imagery.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
Why not using a nix-shell and listing all the dependencies?

Reply:
It's used for testing for now and requires the use of the devShell. Can you explain a bit more about what you were thinking should change?